lib/CIndexStoreDB/CIndexStoreDB.cpp

```diff
@@ -95,7 +114,7 @@ indexstoredb_index_create(const char *storePath, const char *databasePath,
   if (auto index =
           IndexSystem::create(storePath, databasePath, libProviderObj, delegate,
                               useExplicitOutputUnits, readonly,
-                              /*enableOutOfDateFileWatching=*/false, listenToUnitEvents, wait,
+                              enableOutOfDateFileWatching, listenToUnitEvents, wait,
```

Comment:
There's a comment about `enableOutOfDateFileWatching` above that is no longer accurate for this part of the code.  If the performance considerations are still relevant, they should probably be mentioned in the doc comment for the `enableOutOfDateFileWatching` parameter you added to `IndexStoreDB.init`.

Reply:
Done.